.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
```

Comment:
We probably don't need to run this on PRs against any branch other than the main one.